.prettierrc

```
@@ -1,5 +1,6 @@
 {
   "semi": false,
   "arrowParens": "always",
-  "singleQuote": true
+  "singleQuote": true,
+  "trailingComma": false
```

```suggestion
  "trailingComma": none
```

I got a warning that "false" was deprecated, so switching it to "none" instead.

---

Yep, though "none" needs quotes around it.
```suggestion
  "trailingComma": "none"
```